<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>252</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
      background-color:gray;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 20px;
    padding: 5px;
    qproperty-alignment: 'AlignRight';
}
QLabel::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QLabel::pressed{
    background-color: gray;  
    border: 2px solid black;  
    
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_posneg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multi">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>120</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;
    background-color: black;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: black;
    background-color: white;
    font-size: 20px;
    border: 2px solid #555555;
    border-radius: 15px;
    padding: 5px;
    qproperty-alignment: 'AlignCenter';
    margin:5px;
}
QPushButton::hover{
    background-color: #2e2e2e;  
    border: 2px solid white;  
}
QPushButton::pressed{
    color:white;
    background-color: black;  
    border: 2px solid white;  
    border-radius: 10;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
